wordfreq/__init__.py

```diff
@@ -240,4 +249,12 @@ def random_words(lang='en', wordlist='combined', nwords=4, bits_per_word=12,
 
 def random_ascii_words(lang='en', wordlist='combined', nwords=4,
                        bits_per_word=12):
+    """
+    Returns a string of random, space separated, ascii words.
+
+    These words are are of the given language and from the given wordlist.
+    There are a total of nwords words in the string.
+    bits_pre_words is an estimate of the entropy provided by each word.
```

Comment:
This should say "bits_per_word".

The line "If ascii_only, then the words are ascii only" in this docstring is unnecessary, because this one always restricts it to ASCII words. It's also a bit too literal-minded to make for a good docstring, even when it appears in `random_words`. I'd suggest removing this one, and changing the `random_words` one to "You can restrict the selection of words to those written in ASCII characters by setting `ascii_only` to True."